### ISSUE TITLE
Simple autoencoder keeping constant tensor as predict in keras

### ISSUE THREAD

**Issue by MohamedAliB** [keras-team/keras#12646]
I'm new in keras and deep learning field. In fact, I want  to make a dense vector for each document in my data so that i built a simple autoencoder using keras library.
The input data are normalized using Word2vec with 200 as embedding size and all features are between -1 and 1. I prepared a 3D tensor that contains 137 samples (number of document) with 469 columns (maximum numbers of words) and the third dimension is the embedding size.I used mse loss function and GRU as recurrent neural network. I am having the same vector for all documents as the autoencoder prediction output while loss start with a very low value and  became constant after a few number of epochs.
I tried different number of epochs but I got the same thing. I tried also to change the batch size but no change. Can any one help me find the problem please.

`input = Input(shape=(469,200))

encoder = GRU(120,activation='sigmoid',dropout=0.2)(input)

neck = Dense(20)(encoder)

decoder1 = RepeatVector(469)(neck)

decoder1 = GRU(120,return_sequences=True,activation='sigmoid',dropout=0.2)(decoder1)

decoder1 = TimeDistributed(Dense(200,activation='tanh'))(decoder1)

model = Model(inputs=input, outputs=decoder1)

model.compile(optimizer='adam', loss='mse')

history = model.fit(x_train, x_train,validation_data=(x_test,x_test) ,epochs=10, batch_size=8)`

this is a part of x_train "input data" : 

![x_train](https://user-images.githubusercontent.com/48586690/55807421-17317b80-5ada-11e9-9c20-af5523d1bf55.png)

print(model.predict(x_train)) return this values (same vectors):
![predict_x_train](https://user-images.githubusercontent.com/48586690/55807505-37f9d100-5ada-11e9-962f-f380e29e9157.png)
loss: 
![courbe_loss](https://user-images.githubusercontent.com/48586690/55807537-4811b080-5ada-11e9-8d13-6cef53d5beba.png)
![Capture d’écran de 2019-04-08 12-23-30](https://user-images.githubusercontent.com/48586690/55807538-4811b080-5ada-11e9-9e44-246515e09eb4.png)
![Capture d’écran de 2019-04-08 12-23-18](https://user-images.githubusercontent.com/48586690/55807539-48aa4700-5ada-11e9-8490-21e01bc82c58.png)
![Capture d’écran de 2019-04-08 12-14-50](https://user-images.githubusercontent.com/48586690/55807540-4942dd80-5ada-11e9-80a8-15b61a7382eb.png)
![Capture d’écran de 2019-04-08 12-14-33](https://user-images.githubusercontent.com/48586690/55807541-4942dd80-5ada-11e9-99ae-d1b1b2ef37fc.png)

Why "model.predict(x_train)" return the same vector for the 137 samples ?

"I am working on text reconstruction and i'am using word2vec to generate embedding vector for each word."

Thank you in advance.

**Comment by jvishnuvardhan** [comment]
@MohamedAliB Which backend you are using?
Why are you same (x_train) as input and output in this model.fit line
history = model.fit(x_train, x_train,validation_data=(x_test,x_test) ,epochs=10, batch_size=8)`

Thanks
